From 4f9bae4246cc69dac3cdd844fd7b16c0889f1fac Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Wed, 15 Mar 2017 17:23:32 +0800
Subject: [PATCH] FROMLIST: arm64: dts: rockchip: rk3368 swap clust0 and clust1

Before this patch, clust1 has little core0~3, clust0 has big core0~3
cpu_l | cpu0 |
cpu_l | cpu1 | clust1
cpu_l | cpu2 |
cpu_l | cpu3 |
----------------------
cpu_b | cpu4 |
cpu_b | cpu5 | clust0
cpu_b | cpu6 |
cpu_b | cpu7 |

With this patch, clust0 will have little core0~3, clust1 will have big core0~3
cpu_l | cpu0 |
cpu_l | cpu1 | clust0
cpu_l | cpu2 |
cpu_l | cpu3 |
----------------------
cpu_b | cpu4 |
cpu_b | cpu5 | clust1
cpu_b | cpu6 |
cpu_b | cpu7 |

It makes no other change, just keep same with other SoCs definations.

Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
(am from https://patchwork.kernel.org/patch/9625109/)

Change-Id: I1beea4d3e75409d3a1f1614b0b86f1a929db4eee
---
 arch/arm64/boot/dts/rockchip/rk3368.dtsi | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index b86151955790..fd06e6ca7c19 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -80,31 +80,31 @@
 		cpu-map {
 			cluster0 {
 				core0 {
-					cpu = <&cpu_b0>;
+					cpu = <&cpu_l0>;
 				};
 				core1 {
-					cpu = <&cpu_b1>;
+					cpu = <&cpu_l1>;
 				};
 				core2 {
-					cpu = <&cpu_b2>;
+					cpu = <&cpu_l2>;
 				};
 				core3 {
-					cpu = <&cpu_b3>;
+					cpu = <&cpu_l3>;
 				};
 			};
 
 			cluster1 {
 				core0 {
-					cpu = <&cpu_l0>;
+					cpu = <&cpu_b0>;
 				};
 				core1 {
-					cpu = <&cpu_l1>;
+					cpu = <&cpu_b1>;
 				};
 				core2 {
-					cpu = <&cpu_l2>;
+					cpu = <&cpu_b2>;
 				};
 				core3 {
-					cpu = <&cpu_l3>;
+					cpu = <&cpu_b3>;
 				};
 			};
 		};
-- 
2.35.3

